src/main/java/Parser.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Parser helps to extract the information in txt files stored in strings into
+ * a desired format.
+ */
+public class Parser {
+
+    /**
+     * Takes in a line from the txt file and outputs the description of a Todo object.
+     * @param str The line read in from the txt file.
```

Comment:
Should have an empty line between description and parameters.

Reply:
Noted. Thank you for the feedback.